### PR TITLE
Revert "433 detect url in goal parser"

### PR DESCRIPTION
**Pull request by tijlleenders**
Reverts tijlleenders/ZinZen#540

Had to do this as this PR is causing the AddGoal page to become unavailable.

### CODE REVIEW

File: src/components/GoalsComponents/GoalsForm.tsx [hunk 1]
```diff
@@ -141,17 +134,6 @@ export const GoalsForm: React.FC<GoalsFormProps> = ({ selectedColorIndex, parent
...
           }
-          className={urlDetection() !== "" ? "duration" : "blank"}
-        >
-          {urlDetection()}
-        </button>
-        <button
-          type="button"
-          style={
-            darkModeStatus
-              ? { backgroundColor: darkColors[selectedColorIndex] }
```
Comment (Tushar-4781) [inline src/components/GoalsComponents/GoalsForm.tsx:152]: color pallet has to be used

**Comment by GithinjiHans** [comment]
I think I should make a PR to correct this instead of reverting the whole PR

**Comment by Tushar-4781** [comment]
@GithinjiHans i corrected your PR back. No need now. We are gtg now :D

**Comment by tijlleenders** [comment]
Ok, so @GithinjiHans changes are not reverted by this PR, just corrected with the fix from @Tushar-4781 

Thanks!

image

**Review by Tushar-4781 — APPROVED** [review]

**Review by GithinjiHans — APPROVED** [review]

**Pull request merged**